index.js

```diff
@@ -1,125 +1,127 @@
-import { fileTypeFromBuffer } from 'file-type';
+import {fileTypeFromBuffer} from 'file-type';
 import fs from 'fs';
 import OpenAI from 'openai';
 
 async function main() {
-  let images = []
-  if (process.env.IMAGES) {
-    images = process.env.IMAGES.split(/(?<!\\),/) // this matches all unescaped commas
-    for (let i = 0; i < images.length; i++) {
-      images[i] = images[i].replace(/\\,/g, ',')
+    let images = []
+    if (process.env.IMAGES) {
+        images = process.env.IMAGES.split(/(?<!\\),/) // this matches all unescaped commas
+        for (let i = 0; i < images.length; i++) {
+            images[i] = images[i].replace(/\\,/g, ',')
+        }
     }
-  }
-
-  if (images.length === 0) {
-    console.log("No images provided. Please provide a list of images to send to the vision model.")
-    process.exit(1)
-  }
-
-  const prompt = process.env.PROMPT ?? ""
-  if (prompt === "") {
-    console.log("No prompt provided. Please provide a prompt to send to the vision model.")
-    process.exit(1)
-  }
-
-  const options = {}
-
-  if (process.env.OPENAI_API_KEY) {
-    options.openaiApiKey = process
-  }
-  if (process.env.OPENAI_BASE_URL) {
-    options.baseUrl = process.env.OPENAI_BASE_URL
-  }
-  if (process.env.OPENAI_ORG_ID) {
-      options.orgId = process.env.OPENAI_ORG_ID
-  }
-
-  if (process.env.MAX_TOKENS) {
-    options.maxTokens = parseInt(process.env.MAX_TOKENS)
-  } else {
-    options.maxTokens = 2048
-  }
-
-  if (process.env.MODEL) {
-    if (process.env.MODEL === 'gpt-4-turbo') {
-      options.model = 'gpt-4-turbo'
-    } else if (process.env.MODEL === 'gpt-4o') {
-      options.model = 'gpt-4o'
-    } else {
-        console.log("Invalid model provided. Please provide a valid model.")
+
+    if (images.length === 0) {
+        console.log("No images provided. Please provide a list of images to send to the vision model.")
+        process.exit(1)
+    }
+
+    const prompt = process.env.PROMPT ?? ""
+    if (prompt === "") {
+        console.log("No prompt provided. Please provide a prompt to send to the vision model.")
         process.exit(1)
     }
-  } else {
-    options.model = 'gpt-4o'
-  }
-
-  if (process.env.DETAIL) {
-    if (process.env.DETAIL === 'low') {
-      options.detail = 'low'
-    } else if (process.env.DETAIL === 'high') {
-      options.detail = 'high'
-    } else if (process.env.DETAIL === 'auto') {
-      options.detail = 'auto'
+
+    const options = {}
+
+    if (process.env.OPENAI_API_KEY) {
+        options.openaiApiKey = process
```

Comment:
I noticed this was like this before your changes too...not sure how this was working, lol.

```suggestion
        options.openaiApiKey = process.env.OPENAI_API_KEY
```